db/migrate/20200411005517_add_status_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrders < ActiveRecord::Migration[5.1]
```

Comment:
Nice! You should of already had access to this so might need to have Mariana pull master asap since she is working on US 26, which is when this should of been implemented.

Reply:
I don't see it in master and there are no MC branches pushed up to GH right now.

Reply:
Correct. We finished US25 last night and then I helped set up the test for 26 which is where that migration/association takes place. I don't know if she finished or not. She won't be available until 5 today. Hoping there isn't any isssues with merge conflicts and db